docker-compose.yaml

```diff
@@ -171,7 +171,7 @@ services:
     depends_on:
       pyciemss-api:
         condition: service_started
-  simulation-service:
+  sciml-service:
```

Comment:
If you change this, you need to update the `depends_on` and any other reference if there is one.